scripts/docker/official/nginx-oss-with-nginx-agent/alpine/Dockerfile

```
@@ -0,0 +1,21 @@
+ARG NGINX_VERSION=1.25.1-alpine
+FROM nginx:${NGINX_VERSION}
+LABEL maintainer="NGINX Agent Maintainers <agent@nginx.com>"
+
+RUN set -x \
+    && apk add curl bash iproute2
+
+RUN printf "%s%s\n" \
+    "http://packages.nginx.org/nginx-agent/alpine/v3.16" \
+    "/main" | tee -a /etc/apk/repositories
```

Makes sense to do something like the following? 

So when `ARG NGINX_VERSION` is modified and includes a alpine version change, the correct agent version will always be installed?

```suggestion
RUN printf "%s%s%s\n" \
    "http://packages.nginx.org/nginx-agent/alpine/v" \
    `grep -o -E '^[0-9]+\.[0-9]+' /etc/alpine-release` \
    "/main" | tee -a /etc/apk/repositories
```